### PR TITLE
Init project

### PR DESCRIPTION
**Pull request by rashidfirhat98**
ready

### CODE REVIEW

**Comment by rashidfirhat98** [comment]
ready

File: config/credentials.yml.enc [hunk 1]
```diff
@@ -1 +1,5 @@
-n+W2+C0Fbd1M/83dhFlJ+aPHWRrotlFUyaAUcYUgG7ybwYEc+uKoDIbjFeUNr+PsQxFO5RLsQOowI8DsEwjJEOj3yCunYsgtTNpjrhB1f2C2gAy1OlqQGk9l0gBcq4XOBoRdKR5ZiBQT2KRdvdZv2tuJwWYnd/N0SxJZkjTnFTn2EI7+9ggE2JM0dMdhjl02Xvvh3ZxnOowlSG1kO8OvrHyFEKvYsIPdrBZ18Te0Wpcrc5RQy9EpZP4K+c7MZ7n0Cd3CaeFxwg2xHcdya1dJEv7KgPMqhykOVC8DcsC6ZuZSW45OYZBudSHQQUX3q+UcQnO5+olvk/gJAtN8RphngjLyP+kJgfqKYoJDxCvgdt7onK3VjJZ2xkiu6E8H8MaLyIaUBgi3qtcG8ot2IKdfDE6Sa43SNPrvS+qk--JM7klsJFje1tC9wd--92EuGjYjAT85H5Gdj3/J8g==
\ No newline at end of file
+<<<<<<< HEAD
+IGDLp7T2EcrEirhgX3BS1ijUIE92joKe8Im/P1ZqKui61mhfcdlHpdjIdjNRCTd2LvHZYfUJ+n+HSD+nW2JrNVfMuLDleD2BLYyp1lfvqrtVnPHtjd715Lp4nt4AL47rdLepMp401rIIGi+iLala1LWu2H1+3WoWBcfne+/20Hv0vDhsElqBg8ZzKJz7dOei6IN+Vo+CeoUX15dtlAWquHG2MKTePhK0dESSiGXBdNIqGDiNVNCl30rxksVQsV850eyQ93+HlV81baMUIQd6Y7bcuvq2YYlO3rqIXrc3S18QfU43Afm4mH9qn//dxT670JhV3I7/38fqJgWo9eOmcFRnOZpkeM/4NBj00rKmx08ZIjbIpi4AwQal2mTo59XMKTH1OzMIDvGrcFQsudYXQe2+vIr3uBLEwrAu--AvVshDM2kMl4j+F1--n81cDqdFy+wsmW2YeE7g4g==
+=======
```
Comment (ngocvt-0484) [inline config/credentials.yml.enc]: please fix conflict

**Comment by ngocvt-0484** [comment]
please fix conflict and change 6 commits to 1 commit

**Comment by rashidfirhat98** [comment]
ready

File: config/credentials.yml.enc (hunk 1 above)
Comment (rashidfirhat98) [reply config/credentials.yml.enc]: done but im not sure if i did it right

File: Gemfile [hunk 2]
```diff
@@ -1,22 +1,26 @@
...
 
-ruby '2.7.1'
+source "https://rubygems.org"
+git_source(:github){|repo| "https://github.com/#{repo}.git"}
+
+ruby "2.7.1"
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '~> 6.0.3', '>= 6.0.3.2'
+gem "rails", "~> 6.0.3", ">= 6.0.3.2"
```
Comment (saobv-2223) [inline Gemfile]: please, remove all the comment lines

**Comment by rashidfirhat98** [comment]
ready

**Comment by manhnt-0960** [comment]
commented
2 commits, please fix it

File: Gemfile [hunk 3]
```diff
@@ -1,3 +1,48 @@
...
+  gem "webdrivers"
+end
+
+gem "tzinfo-data", platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+
+group :development, :test do
+  gem "rubocop", "~> 0.74.0", require: false
+  gem "rubocop-rails", "~> 2.3.2", require: false
+end
+=======
```
Comment (saobv-2223) [inline Gemfile]: fix conflict, please

File: config/credentials.yml.enc [hunk 4]
```diff
@@ -1 +1,5 @@
-n+W2+C0Fbd1M/83dhFlJ+aPHWRrotlFUyaAUcYUgG7ybwYEc+uKoDIbjFeUNr+PsQxFO5RLsQOowI8DsEwjJEOj3yCunYsgtTNpjrhB1f2C2gAy1OlqQGk9l0gBcq4XOBoRdKR5ZiBQT2KRdvdZv2tuJwWYnd/N0SxJZkjTnFTn2EI7+9ggE2JM0dMdhjl02Xvvh3ZxnOowlSG1kO8OvrHyFEKvYsIPdrBZ18Te0Wpcrc5RQy9EpZP4K+c7MZ7n0Cd3CaeFxwg2xHcdya1dJEv7KgPMqhykOVC8DcsC6ZuZSW45OYZBudSHQQUX3q+UcQnO5+olvk/gJAtN8RphngjLyP+kJgfqKYoJDxCvgdt7onK3VjJZ2xkiu6E8H8MaLyIaUBgi3qtcG8ot2IKdfDE6Sa43SNPrvS+qk--JM7klsJFje1tC9wd--92EuGjYjAT85H5Gdj3/J8g==
\ No newline at end of file
+
```
Comment (saobv-2223) [inline config/credentials.yml.enc]: image

**Comment by rashidfirhat98** [comment]
ready

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>SampleApp</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
```
Comment (saobv-2223) [inline app/views/layouts/application.html.erb]: using " instead of '

File: Gemfile [hunk 6]
```diff
@@ -0,0 +1,44 @@
+source "https://rubygems.org"
+git_source(:github){|repo| "https://github.com/#{repo}.git"}
+
+ruby "2.7.1"
+
+gem "rails", "~> 6.0.3", ">= 6.0.3.2"
```
Comment (saobv-2223) [inline Gemfile]: image

**Comment by rashidfirhat98** [comment]
ready

**Comment by manhnt-0960** [comment]
conflicted

**Comment by ngocvt-0484** [comment]
please fix conflict
`
git checkout master
git pull origin master
git checkout init_project
git rebase master

after fix conflict

git add .
git commit --amend --no-edit
git push origin init_project -f`

**Pull request closed without merging**